### PR TITLE
Add SkipLagged API & Pokemon id in Pokedex

### PR DESCRIPTION
**Pull request by cicklow**
sin titulo

### CODE REVIEW

File: PokemonGo.RocketAPI.Console/GUI.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public GUI()
 
         private void GUI_Load(object sender, EventArgs e)
         {
-            MessageBox.Show("The Bot isn't done! Be aware that you can get banned!");
+            //MessageBox.Show("The Bot isn't done! Be aware that you can get banned!");
             _clientSettings = new Settings();
```
Comment (SscSPs) [inline PokemonGo.RocketAPI.Console/GUI.cs]: Why?

**Comment by Ar1i** [comment]
Would be nice if you can fix the conflicts

File: PokemonGo.RocketAPI.Console/GUI.cs (hunk 1 above)
Comment (Logxn) [inline PokemonGo.RocketAPI.Console/GUI.cs]: Honestly, every time I test a function and I start the bot, this messagebox gets on my fucking nerves. I guess he commented it out for testing purposes and forgot to un-comment it

Comment (suljo94) [inline PokemonGo.RocketAPI.Console/GUI.cs]: @Logxn YOU could display the warning in the main gui instead of the pop up box,  less hassle for everyone

**Comment by cicklow** [comment]
@Ar1i hello, conflicts?

**Comment by Logxn** [comment]
Hahahaha xD I will merge it

File: PokemonGo.RocketAPI.Console/GUI.cs (hunk 1 above)
Comment (Logxn) [inline PokemonGo.RocketAPI.Console/GUI.cs]: @suljo94 

Nah they wouldnt even read it

Comment (suljo94) [inline PokemonGo.RocketAPI.Console/GUI.cs]: Make it a agreement style check box that saves after 1x clicking it

**Pull request merged**